test/src/dao/RestDAO.js

```diff
@@ -100,13 +100,14 @@ describe('RestDAO', function() {
             } catch (err) {
               return Promise.resolve(createResponse({ status: 500 }));
             }
+            var sink = data.sink || this.ArraySink.create();
```

Comment:
Isn't this redundant? Won't all DAO's treat an undefined sink as an ArraySink anyway?